### PR TITLE
piper: update voices

### PR DESCRIPTION
**Pull request by joshuadavidson**
The following voices are present and can be selected in an voice assist pipeline, but cannot be selected in the Piper addon for default use in tts service calls:
* `cy_GB-gwryw_gogleddol-medium`
* `en_US-bryce-medium`
* `en_US-john-medium`
* `en_US-norman-medium`
* `it_IT-paola-medium`

## Summary by CodeRabbit

- **New Features**
	- Introduced several new voice options, including "gwryw_gogleddol," "bryce," "john," "norman," and "paola," to enhance the text-to-speech experience.
- **Updates**
	- Updated the text-to-speech system version from 1.5.1 to 1.5.2, reflecting the latest improvements and features.
- **Documentation**
	- Updated the changelog to include details about the new voice options in version 1.5.2.

### CODE REVIEW

**Comment by agners** [comment]
Thanks for your PR! :pray: 

> The `en_US-norman-medium` voice is present and can be selected in an voice assist pipeline, but cannot be selected in the Piper addon for default use in tts service calls.

From what I can tell norman got added by this commit: https://github.com/rhasspy/piper/commit/40c852d339fc7532cbbf28a4a942a7c96c185c96. Can you also add the others which got added there?

Can you also add

**Comment by joshuadavidson** [comment]
Thanks for this commit reference! I was looking for the source of truth to get them all in one go. I'll update.

**Comment by joshuadavidson** [comment]
@coderabbitai full review

**Comment by joshuadavidson** [comment]
@coderabbitai summary

File: piper/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.5.2
+
+- Add voice for English: norman
```
Comment (agners) [inline piper/CHANGELOG.md:5]:
```suggestion
- Add missing voices supported by piper (bryce, john, norman and paola)
```

**Review by agners — APPROVED** [review]

**Pull request merged**